src/ChurchCRM/model/ChurchCRM/Person.php

```diff
@@ -651,9 +651,7 @@ public function getAge(?\DateTimeInterface $now = null): ?string
         if (!$birthDate instanceof \DateTimeImmutable || $this->hideAge()) {
             return false;
         }
-        if (!$now instanceof \DateTimeInterface) {
-            $now = new \DateTimeImmutable('today');
-        }
+        $now = ($date == null) ? new \DateTimeImmutable('today') : \DateTimeImmutable::createFromFormat('Y-m-d', $date);
```

Comment:
I think the parenthesis are redundant here, so it can be removed :)

```suggestion
        $now = $date == null ? new \DateTimeImmutable('today') : \DateTimeImmutable::createFromFormat('Y-m-d', $date);
```

Reply:
Sure